packages/fxa-content-server/app/scripts/templates/legal.mustache

```diff
@@ -6,5 +6,6 @@
   <section class="links">
     <a href="/legal/terms" class="left">{{#t}}Terms of Service{{/t}}</a>
     <a href="/legal/privacy" class="right">{{#t}}Privacy Notice{{/t}}</a>
+    <a href="/legal/subscription_terms" class="left">{{#t}}Subscription Platform Terms of Service{{/t}}</a>
```

Comment:
Hmm, this link isn't in mockups, we *might* need to get this placement and text run past UX / (ironically) legal.

Reply:
Good point.  I can just remove it.  It'd affect a couple functional tests though, since this is the only place that link to the new page at the moment.

@clouserw What's your take?

Reply:
Let's not add any links to this for now.  Legal needs to finish the content for it and then they'll tell us what to change.  I'll file a bug for that.